synapse/storage/database.py

```diff
@@ -1125,7 +1125,7 @@ async def simple_select_one_onecol(
         self,
         table: str,
         keyvalues: Dict[str, Any],
-        retcol: Iterable[str],
+        retcol: str,
```

Comment:
I think you want this on the overload(s) of `simple_select_one_onecol` above this too.

Reply:
I think the IDE elided these…? But yes, I spotted them now

Reply:
I think the IDE elided these…? But yes, I spotted them now